mackerel-plugin-snmp/README.md

```diff
@@ -7,10 +7,19 @@ SNMP V2c custom metrics plugin for mackerel.io agent.
 can specify multiple metric-definitions in the form of `OID:NAME[:DIFF?][:STACK?]` args.
```

Comment:
btw, Is it ok not to change this usage?

Reply:
Yes. this is new part of command arguments. when Not specified, No change in behavior.